src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+}
+
+.stars__star2 {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__star,
+.stars__star2 {
+  background-position: center;
+  background-size: cover;
```

Comment:
try to remove hardcoded width above and remove background-size rule and rerun tests